core-bundle/src/Image/ImageFactory.php

```diff
@@ -112,6 +112,7 @@ public function setPredefinedSizes(array $predefinedSizes): void
 
     public function create($path, $size = null, $options = null): ImageInterface
     {
+        /** @var mixed $options */
```

Comment:
Really? I figured `mixed` was the implied content type?

Reply:
It's `int|array|ResizeConfiguration|null` (via `ImageFactoryInterface`). 
see https://github.com/contao/contao/blob/d8ae1290c620082105a0951a4f249c87590a3230/core-bundle/src/Image/ImageFactoryInterface.php#L26

We either must trust the annotation or remove the check. (Or we configure PHPStan to mistrust the annotated types - IIRC there is an option like this).

Reply:
Mistrusting the annotated types of method parameters seems reasonable to me.

Reply:
As mistrusting types via config unfortunately [doesn't work](https://github.com/contao/contao/pull/1924#discussion_r453186771) in this case - what shall we do here?
1) explicitly mark parameter as uncertain `/** @var mixed $options */`
1) ignore checking for this statement `/** @phpstan-ignore-next-line */`
1) add exception to `phpstan.neon.dist`

I'd actually prefer the first option.